<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewProjectDialog</class>
 <widget class="QDialog" name="NewProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New slowmo Project</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLineEdit" name="projectDir"/>
   </item>
   <item row="2" column="1" colspan="5">
    <widget class="QPushButton" name="browseProjectDir">
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="16" column="0" colspan="7">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="bAbort">
       <property name="text">
        <string>Abort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bOk">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="6">
    <widget class="QCheckBox" name="cbDirectoryCreated">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Directory will be created</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="11" column="0" rowspan="2" colspan="7">
    <widget class="QGroupBox" name="groupVideo">
     <property name="title">
      <string>Video source</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QLineEdit" name="inputVideo"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="browseInputVideo">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblcVideoInfo">
        <property name="text">
         <string>Video information:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPlainTextEdit" name="txtVideoInfo">
        <property name="baseSize">
         <size>
          <width>600</width>
          <height>0</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblcInputVideo">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Input video</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="13" column="0" colspan="7">
    <widget class="QGroupBox" name="groupImages">
     <property name="title">
      <string>Image source</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="lblcInputImages">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Input images</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="browseInputImages">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblcImageInfo">
        <property name="text">
         <string>Images information:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QListWidget" name="inputImages"/>
      </item>
      <item row="4" column="0">
       <widget class="QLineEdit" name="txtImageInfo">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="0" colspan="7">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="radioVideo">
       <property name="text">
        <string>Video file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioImages">
       <property name="text">
        <string>Images</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="15" column="0" colspan="7">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblcProjectDir">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Project Directory</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblcProjectFilename">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Project Filename</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLineEdit" name="projectFilename"/>
   </item>
   <item row="6" column="1" colspan="5">
    <widget class="QLabel" name="lblcFilenameExtension">
     <property name="text">
      <string>.sVproj</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>You should preferredly use an empty directory here. Each project needs its own project directory.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="6">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>The project will be saved to this file right after confirming this dialog.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>projectDir</tabstop>
  <tabstop>browseProjectDir</tabstop>
  <tabstop>cbDirectoryCreated</tabstop>
  <tabstop>projectFilename</tabstop>
  <tabstop>radioVideo</tabstop>
  <tabstop>radioImages</tabstop>
  <tabstop>inputVideo</tabstop>
  <tabstop>browseInputVideo</tabstop>
  <tabstop>inputImages</tabstop>
  <tabstop>browseInputImages</tabstop>
  <tabstop>txtImageInfo</tabstop>
  <tabstop>bAbort</tabstop>
  <tabstop>bOk</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
